app/controllers/events_controller.rb

```diff
@@ -5,6 +5,8 @@ class EventsController < ApplicationController
   before_action :authorize_event_author, only: [:update, :destroy]
 
   def index
+    @events = @project.events
+    @project_id = @project.id
```

Comment:
you don;t need to extract id, when you pass an object rails will know what to do